infrastructure/templates/public-api/components/alerts/containerApps/restarts.bicep

```diff
@@ -0,0 +1,33 @@
+import { Severity } from '../types.bicep'
+
+@description('Names of the resources that these alerts are being applied to.')
+param resourceNames string[]
+
+@description('The alert severity.')
+param severity Severity = 'Warning'
+
+@description('Name of the Alerts Group used to send alert messages.')
+param alertsGroupName string
+
+@description('Tags with which to tag the resource in Azure.')
+param tagValues object
+
+module alerts '../staticMetricAlert.bicep' = [for name in resourceNames: {
+  name: '${name}RestartsAlertModule'
+  params: {
+    alertName: '${name}-restarts'
+    resourceIds: [resourceId('Microsoft.App/containerApps', name)]
+    resourceType: 'Microsoft.App/containerApps'
+    query: {
+      metric: 'RestartCount'
+      aggregation: 'Average'
```

Comment:
Great idea!  Updated to "Total" because that's what the doco suggests is supported.

![image](https://github.com/user-attachments/assets/09eacebc-7a74-4dbc-96cc-b99dd39db7c5)